### PR TITLE
api endpoints updated

### PR DESCRIPTION
**Pull request by pondrejk**
all changes tied to BZ#2159105

### CODE REVIEW

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/endtoend/test_api_endtoend.py -k links

**Review by adarshdubey-star — APPROVED** [review]

**Comment by pondrejk** [comment]
well, there are changes in stream hence prt failure

File: tests/foreman/endtoend/test_api_endtoend.py
Comment (shweta83) [inline tests/foreman/endtoend/test_api_endtoend.py:852]: I don't see product sync api here. Did we miss it here?

**Review by jyejare — COMMENTED** [review]
@pondrejk This change in stream is yet to be snapped or yet to be snapped? 

Because the PRT runs on the `stream satellite image` when the branch master and if PRT is failing then its either stream image is not latest or PRT have catch the real issue here  ?

File: tests/foreman/endtoend/test_api_endtoend.py
Comment (pondrejk) [reply tests/foreman/endtoend/test_api_endtoend.py:852]: Well, it was removed intentionally in 2159105, some more notes in https://github.com/Katello/katello/pull/10632 
I'm not owning that bz, so if the future qe contact of that bz negotiates different changes I'd be happy to adjust this test again

**Comment by pondrejk** [comment]
@jyejare to clarify, I'm addressing 6.14 changes here, I want it to be cherrypicked trough master so it is not missing there, but there are more changes in stream that I'm not addressing here, hence the prt failure

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/endtoend/test_api_endtoend.py -k links
env:
  ROBOTTELO_server__deploy_arguments__deploy_sat_version: '6.14.0'

**Comment by Gauravtalreja1** [comment]
```
trigger: test-robottelo
pytest: tests/foreman/endtoend/test_api_endtoend.py -k links
env:
    ROBOTTELO_server__deploy_arguments__deploy_sat_version: '6.14.0'
```

**Comment by Gauravtalreja1** [comment]
```
trigger: test-robottelo
pytest: tests/foreman/endtoend/test_api_endtoend.py -k links
env:
    ROBOTTELO_server__deploy_arguments__deploy_sat_version: '6.14.0'
    ROBOTTELO_server__version__snap: '7.0'
```

**Comment by Gauravtalreja1** [comment]
```
trigger: test-robottelo
pytest: tests/foreman/endtoend/test_api_endtoend.py -k links
env:
    ROBOTTELO_server__deploy_arguments__deploy_sat_version: '6.14.0'
    ROBOTTELO_server__deploy_arguments__deploy_snap_version: '7.0'
```

File: tests/foreman/endtoend/test_api_endtoend.py
Comment (lhellebr) [reply tests/foreman/endtoend/test_api_endtoend.py:852]: I see only one of the endpoints in the BZ got changed in the end, the other two were kept. Is that intended?

File: tests/foreman/endtoend/test_api_endtoend.py
Comment (lhellebr) [inline tests/foreman/endtoend/test_api_endtoend.py:915]: I don't think this endpoint has anything to do with that BZ.

**Review by lhellebr — APPROVED** [review]
Two comments to make double sure this is the change you want, but LGTM if you confirm.

**Review by Gauravtalreja1 — APPROVED** [review]
Ack, merge pending on @lhellebr comments

File: tests/foreman/endtoend/test_api_endtoend.py
Comment (pondrejk) [reply tests/foreman/endtoend/test_api_endtoend.py:852]: @shweta83 the removed endpoint was about monitoring the sync status, product sync api is still there under  `/katello/api/products/:id/sync`

Comment (pondrejk) [reply tests/foreman/endtoend/test_api_endtoend.py:852]: @lhellebr hm, you're right, I'll check with the QA owner of that bz

File: tests/foreman/endtoend/test_api_endtoend.py
Comment (pondrejk) [reply tests/foreman/endtoend/test_api_endtoend.py:915]: Ah, true, I believe this is a feature, introduced with https://github.com/theforeman/foreman_webhooks/pull/57 (don't have the matching bz at hand right now)

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**
